kubeflow/kubeflux/all.libsonnet

```diff
@@ -0,0 +1,283 @@
+          {
+            "apiGroups": [
+              "*"
+            ],
+            "resources": [
+              "*"
+            ],
+            "verbs": [
+              "*"
+            ]
+          },
+          {
+            "nonResourceURLs": [
+              "*"
+            ],
+            "verbs": [
+              "*"
+            ]
+          }
+        ]
+      },
+
+      roleBinding:: {
+        "apiVersion": "rbac.authorization.k8s.io/v1beta1",
+        "kind": "ClusterRoleBinding",
+        "metadata": {
+          "labels": {
+            "name": "flux",
+          },
+          "name": "flux"
+        },
+        "roleRef": {
+          "apiGroup": "rbac.authorization.k8s.io",
+          "kind": "ClusterRole",
+          "name": "flux"
+        },
+        "subjects": [
+          {
+            "kind": "ServiceAccount",
+            "name": "flux",
+            "namespace": namespace
+          }
+        ]
+      },
+
+    secret:: {
+      // K8s Deployment,
+      "apiVersion": "v1",
+      "kind": "Secret",
+      "metadata": {
+        "name": "flux-git-deploy",
+      },
+      "type": "Opaque"
+    },
+
+    flux:: {
+      // K8s Deployment,
+      "apiVersion": "apps/v1beta1",
+      "kind": "Deployment",
+      "metadata": {
+        "name": "flux",
+      },
+      "spec": {
+        "replicas": 1,
+        "strategy": {
+          "type": "Recreate"
+        },
+        "template": {
+          "metadata": {
+            "labels": {
+              "name": "flux"
+            }
+          },
+          "spec": {
+            "containers": [
+              {
+                "args": [
+                  "--ssh-keygen-dir=/var/fluxd/keygen",
+                  "--git-url="+giturl,
+                  "--git-branch=master"
+                ],
+                "image": "quay.io/weaveworks/flux:1.4.2",
+                "imagePullPolicy": "IfNotPresent",
+                "name": "flux",
+                "ports": [
+                  {
+                    "containerPort": 3030
+                  }
+                ],
+                "volumeMounts": [
+                  {
+                    "mountPath": "/etc/fluxd/ssh",
+                    "name": "git-key",
+                    "readOnly": true
+                  },
+                  {
+                    "mountPath": "/var/fluxd/keygen",
+                    "name": "git-keygen"
+                  }
+                ]
+              }
+            ],
+            "serviceAccount": "flux",
+            "volumes": [
+              {
+                "name": "git-key",
+                "secret": {
+                  "defaultMode": 256,
+                  "secretName": "flux-git-deploy"
+                }
+              },
+              {
+                "emptyDir": {
+                  "medium": "Memory"
+                },
+                "name": "git-keygen"
+              }
+            ]
+          }
+        }
+      }
+    },
+
+    nodeport:: {
```

Comment:
Why do we have two different services?

Reply:
It may just be a remnant from the conversion of Flux from yaml to ksonnet. Removed